packages/gnome_nibbles.rb

```diff
@@ -1,45 +1,34 @@
 require 'package'
 
 class Gnome_nibbles < Package
+  @_commit = '62964e9256fcac616109af874dbb2bd8342a9853'
   description 'snake game, up to four players'
   homepage 'https://wiki.gnome.org/Apps/Nibbles'
-  @_app = File.basename(__FILE__, '.rb').tr('_', '-')
-  @_fullver = '3.38.2'
-  @_mainver = @_fullver.rpartition('.')[0]
-  @_url = "https://download.gnome.org/sources/#{@_app}/#{@_mainver}/#{@_app}-#{@_fullver}"
-  version @_fullver
+  version '3.38.2+git+' + @_commit[0..8]
   license 'GPL-3+ and CC-BY-SA-3.0'
   compatibility 'all'
-  source_url "#{@_url}.tar.xz"
-  source_sha256 `curl -Ls #{@_url}.sha256sum | tail -n1 | cut -d ' ' -f1`.chomp
+  source_url "https://gitlab.gnome.org/GNOME/gnome-nibbles/-/archive/#{@_commit}/gnome-nibbles-#{@_commit}.tar.gz"
+  source_sha256 'ebf93903d36ae939f9ae56f47bc0dea151a60dfe2c63962af84c517ef1d7aba4'
 
   depends_on 'clutter_gtk'
   depends_on 'gsound'
   depends_on 'librsvg'
   depends_on 'libgnome_games_support'
+  depends_on 'gui_setup'
 
   def self.build
-    system "meson #{CREW_MESON_OPTIONS} builddir"
+    system "meson #{CREW_MESON_FNO_LTO_OPTIONS} builddir"
     system 'meson configure builddir'
     system 'ninja -C builddir'
   end
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
-    @_wrapper = <<~EOF
-      #!/bin/sh
-      WAYLAND_DISPLAY=wayland-0
-      GDK_BACKEND=wayland
-      CLUTTER_BACKEND=wayland
-      exec #{@_app}.elf "$@"
-    EOF
-    File.write(@_app, @_wrapper)
-    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/#{@_app}", "#{CREW_DEST_PREFIX}/bin/#{@_app}.elf"
-    FileUtils.install @_app, "#{CREW_DEST_PREFIX}/bin/#{@_app}", mode: 0755
   end
 
   def self.postinstall
-    system 'gdk-pixbuf-query-loaders --update-cache'
-    system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas"
+    system 'update-mime-database', "#{CREW_PREFIX}/share/mime"
+    system 'gdk-pixbuf-query-loaders', '--update-cache'
+    system 'glib-compile-schemas', "#{CREW_PREFIX}/share/glib-2.0/schemas"
```

Comment:
I assume `gdk_pixbuf`, `glib` and `shared_mime_info` are inherited dependencies?

Reply:
They are the dependencies of gtk3, which is a dependency of libgnome-game-support